### PR TITLE
Pull Request

### PR DESCRIPTION
**Pull request by ARamsey118**
Vim improvements mostly

### CODE REVIEW

File: packages/cli/vim/config/.vimrc [hunk 1]
```diff
@@ -28,7 +28,7 @@ set scrolloff=8                 "Makes cursor stay 8 lines away from the top or
 
 
 "Tabs to spaces
-set tabstop=4 shiftwidth=4 noexpandtab
+set tabstop=4 shiftwidth=4 expandtab
```
Comment (Serubin) [inline packages/cli/vim/config/.vimrc:31]: Ugh. I wonder if this is going to get weird on my end again.

File: packages/cli/htop/config/htoprc [hunk 2]
```diff
@@ -11,7 +11,7 @@ show_thread_names=0
 highlight_base_name=0
 highlight_megabytes=1
 highlight_threads=0
-tree_view=1
+tree_view=0
```
Comment (Serubin) [inline packages/cli/htop/config/htoprc:14]: I disagree with you.

Comment (ARamsey118) [inline packages/cli/htop/config/htoprc:14]: I'm well aware

File: packages/cli/vim/config/.vimrc (hunk 1 above)
Comment (ARamsey118) [inline packages/cli/vim/config/.vimrc:31]: From vimdocs:                

_'expandtab'_ _'et'_ _'noexpandtab'_ _'noet'_
'expandtab' 'et'    boolean (default off)
            local to buffer
            {not in Vi}
    In Insert mode: Use the appropriate number of spaces to insert a
    <Tab>.  Spaces are used in indents with the '>' and '<' commands and
    when 'autoindent' is on.  To insert a real tab when 'expandtab' is
    on, use CTRL-V<Tab>.  See also |:retab| and |ins-expandtab|.
    This option is reset when the 'paste' option is set and restored when
    the 'paste' option is reset.
    NOTE: This option is reset when 'compatible' is set.

Comment (Serubin) [inline packages/cli/vim/config/.vimrc:31]: I realize my solution shouldn't work.... but it does on my machine. I don't do enough python to care anymore. I'll pull as-is

**Comment by Serubin** [comment]
Pulled into develop

**Pull request closed without merging**